### PR TITLE
Feature/tw unknown

### PR DESCRIPTION
**Pull request by Minotaurinmaze**
This is a command that gives the player random effects.

### CODE REVIEW

File: src/com/severalcircles/colors/events/ChatEvent.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 public class ChatEvent implements Listener {
     @EventHandler
     public void onPlayerChat(AsyncPlayerChatEvent event) {
-
+        System.out.println("Chat event triggered");
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/events/ChatEvent.java:13]: I am wondering how this got into this pull but you know

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandUnknown.java  24
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5640746&bid=18470967)

File: src/com/severalcircles/colors/commands/CommandUnknown.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                break;
+            case 20:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.HERO_OF_THE_VILLAGE, 48000, 1));
+                break;
+
+        }
+        return true;
+    }
+    }
+//hehe funny command
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/CommandUnknown.java:104]: Looks like somebody forgot to clean up comments

File: src/com/severalcircles/colors/commands/CommandUnknown.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+
+public class CommandUnknown implements CommandExecutor {
+    @Override
+    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] args) {
+        Player player = (Player) commandSender;
+        int randomNumber = (int) Math.round(Math.random() * 20);
+        System.out.println(randomNumber);
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/CommandUnknown.java:21]: Perhaps we should get rid of this for the final? :eye:

**Review by SeveralCircles — APPROVED** [review]
Looks mostly alright! You got my thumbs pointing towards jesus!

**Review by BeegBrainMo — APPROVED** [review]
Very epic, no problems

File: src/com/severalcircles/colors/commands/CommandUnknown.java (hunk 2 above)
Comment (Minotaurinmaze) [reply src/com/severalcircles/colors/commands/CommandUnknown.java:104]: no i just added that

File: src/com/severalcircles/colors/commands/CommandUnknown.java (hunk 3 above)
Comment (Minotaurinmaze) [reply src/com/severalcircles/colors/commands/CommandUnknown.java:21]: Hmmmmm I don't see the issue having it. Can you explain why we should remove it?

File: src/com/severalcircles/colors/events/ChatEvent.java (hunk 1 above)
Comment (Minotaurinmaze) [reply src/com/severalcircles/colors/events/ChatEvent.java:13]: What is that? I don't remember adding that.

**Pull request merged**